build-system/pr-check.js

```diff
@@ -225,6 +225,11 @@ const command = {
     // Disabled because it regressed. Better to run the other saucelabs tests.
     // execOrDie(`${gulp} test --nobuild --saucelabs --oldchrome --compiled`);
   },
+  runVisualDiffTests: function() {
+    // This must only be run for push builds, since Travis hides the encrypted
+    // environment variables required by Percy during pull request builds.
+    execOrDie(`${gulp} visual-diff`);
```

Comment:
Yea it's cause I noticed that `extractPercyKeys` has a few `process.exit(1)` calls.